### ISSUE TITLE
Support $mock->method() shortcut

### ISSUE THREAD

**Issue by amakhrov** [maxfilatov/phpuaca#14]
PHPUnit's mock object has the `method()` object (implemented [magically](https://github.com/sebastianbergmann/phpunit-mock-objects/blob/master/src/Framework/MockObject/Generator/mocked_class_method.tpl.dist) and included into the interface as [PHPDoc annotation](https://github.com/sebastianbergmann/phpunit-mock-objects/blob/master/src/Framework/MockObject/MockObject.php#L15))

Usage:

```
$mock = $this->getMock(MyClass:class);
$mock->method('someMethod')->willReturn('mocked value');
```

So you see it's just a wrapper over

```
$mock->expects($this->any())->method('someMethod');
```

It would be nice if phpuaca supported this method as well as $mock->expects()->method()

**Comment by amakhrov** [comment]
Actually looks like this feature has been already implemented in this PR: #13
The PR has been merged into `unstable-2`, however, it's not included into any release yet.

@maxfilatov is there any particular plan to release an updated version?

Thanks

**Comment by amakhrov** [comment]
Closing the issue - it's implemented in version 1.3.1
